### PR TITLE
Automatic snapshot commit from seacas at 61ec3b1ec

### PR DESCRIPTION
**Pull request by gsjaardema**
Origin repo remote tracking branch: 'origin/master'
Origin repo remote repo URL: 'origin = https://github.com/gsjaardema/seacas'

At commit:

commit 61ec3b1ec953280dad74d3bc86d07ba09c12b4be
Author:  Greg Sjaardema <gsjaardema@gmail.com>
Date:    Thu May 23 07:27:42 2019 -0600
Summary: IOSS: Fix static name issue on rzansel

* Convesion to using fmt:: library
* A few more strcpy -> copy_string replacements
* Clean up fortran do loops
* fix some gcc-9 warnings
* update to new catch2 version
* added test for cgns reordered fields
* added test for cgns append
* eliminate some float->double->float conversions
* Add cgns-decomp executable to show/test structured cgns decompositions.
* add part-prefix option to ejoin to avoid name collision
* update ejoin offset documentation
* convert some strlen==0 to .empty() calls
* convert some char* to std::string in nem_slice/nem_spread
* improved db summary output in Ioss::Region and io_info
* cleaned up qa output in IOSS applications
* fix cgns/hdf5 hang due to mismatched info output based on nodename

@trilinos/seacas 

## Checklist

- [ ] My commit messages mention the appropriate GitHub issue numbers.
- [x] My code follows the code style of the affected package(s).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [code contribution guidelines](../blob/master/CONTRIBUTING.md) for this project.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] No new compiler warnings were introduced.
- [ ] These changes break backwards compatibility.

### CODE REVIEW

File: packages/seacas/applications/blot/cgi/vdicps.f [hunk 1]
```diff
@@ -1801,8 +1801,9 @@ SUBROUTINE VDIQCO(NUM,INDEX,CLRARY,CLRMOD)
             CALL VBERRH(724,5)
             GOTO 100
          END IF
-           DO 200 IC=1,3
-  200      CLRARY(IC,I)=PCOLS(IC,INDEXN)
+           CLRARY(1,I)=PCOLS(1,INDEXN)
```
Comment (mhoemmen) [inline packages/seacas/applications/blot/cgi/vdicps.f]: `CLRARY(1:3,I)=PCOLS(1:3,INDEXN)` ?

File: packages/seacas/applications/blot/cgi/vdicps.f [hunk 2]
```diff
@@ -3428,43 +3430,44 @@ SUBROUTINE PSTA2C(ASCI,CHARAC)
...
+C     ndigit = number of digits to be produced in string
+C     form (pad left with zeros)
+C     istr = character string of at least ndigit characters
+C     
+C     CALLS            -
+C     
+C     EXIT CONDITIONS  - istr contains decimal-string equivalent of int
+C     (ndigits left-justified in istr)
+C     
+C     NARRATIVE        - This routine modified 10/89  S.L.Thompson
```
Comment (mhoemmen) [inline packages/seacas/applications/blot/cgi/vdicps.f]: Do you need to update this comment?

Comment (gsjaardema) [reply packages/seacas/applications/blot/cgi/vdicps.f]: Probably need to take out all of the "modified ..." type comments at some point.  Is neat to see the names of some of the old-timers who worked on some of these routines in the past...

File: packages/seacas/applications/blot/cgi/vdicps.f (hunk 1 above)
Comment (gsjaardema) [reply packages/seacas/applications/blot/cgi/vdicps.f]: That would work, but I usually stick with fortran-77 in the seacas portions... No real reason other than inertia.

File: packages/seacas/applications/blot/cgi/vdicps.f (hunk 2 above)
Comment (mhoemmen) [reply packages/seacas/applications/blot/cgi/vdicps.f]: @gsjaardema I don't mind seeing the history :-)

**Pull request merged**